openapi/mx_platform_api.yml

```diff
@@ -1954,6 +2282,55 @@ components:
         pagination:
           "$ref": "#/components/schemas/PaginationResponse"
       type: object
+    MonthlyCashFlowResponse:
+      properties:
+        guid: 
+          example: MCF-4e431124-4a29-abf9-f059-ab232ac14dbf
+          type: string
+          description: Unique identifier for the monthly cash flow profile. Defined by MX.
+        user_guid:
+          example: USR-6c83f63c-efcc-0189-3f14-100f0bad378a
+          type: string
+          description: Unique identifier for the user the monthly cash flow profile is attached to. Defined by MX.
+        budgeted_income: 
+          example: 1200.12
+          type: integer
```

Comment:
is this supposed to be a `number` type?

Reply:
Good catch. If I make these changes here than the `docsV2` will be out of sync with this one. Since we treat that one as a source of truth, I think it make sense to have someone clean these up in the docsV2 first then cleanup here. What do you think?

Reply:
Will you create a ticket for that and make sure to call out the need to update this as well?

Reply:
I decided it was just as easy to quickly fix the handful I noticed in `MonthlyCashFlow...` and `SplitTransaction` in both repos. I think we now know which task I loathe more vs. Jira ticket or code in two places. 🤣